workspaces/rbac/plugins/rbac/src/components/RolesList/RolesListToolbar.tsx

```diff
@@ -43,25 +39,28 @@ export const RolesListToolbar = ({
   createRoleAllowed: boolean;
   createRoleLoading: boolean;
 }) => {
-  const classes = useStyles();
   return (
     <div>
       {!createRoleLoading && !createRoleAllowed && (
         <Alert severity="warning" data-testid="create-role-warning">
-          <AlertTitle className={classes.alertTitle}>
+          <AlertTitle
+            sx={{
+              fontWeight: 'bold',
+            }}
+          >
             Unable to create role.
           </AlertTitle>
           To enable create/edit role button, make sure required users/groups are
           available in catalog as a role cannot be created without users/groups
           and also the role associated with your user should have the permission
           policies mentioned{' '}
-          <a
+          <Link
             href="https://github.com/backstage/community-plugins/tree/main/workspaces/rbac/plugins/rbac#prerequisites"
             target="blank"
```

Comment:
```suggestion
            target="_blank"
```

There is also a Link in `@backstage/core-components` that handles this automatically.